TYPO3-Solr/ext-solr
# [TASK] Bootstrap 5.1 adaptions

# What this pr does

Adapts the Solr default templates to be compatible with Bootstrap 5.1

# How to test

Install EXT:solr and check frontend rendering, solr-ddev-site contains examples that can be used.

Fixes: #3112
Related: #3150
